scripts/irods/json_validation.py

```diff
@@ -113,10 +114,11 @@ def get_initial_schema_from_web(schema_uri):
         schema = json.loads(response.content)
     return schema
 
-def get_initial_schema_from_file(schema_uri):
-    with open(schema_uri[7:], 'rt') as f:
-        schema = json.load(f)
-    return schema
+def load_json_schema_from_file(schema_uri):
+    # Skipping the first six characters (i.e. the "file://" prefix) results
+    # in either a relative path or an absolute path. 
+    with open(schema_uri.strip()[6:], 'rt') as f:
```

Comment:
why the change from 7 to 6?    was it ... just wrong before?

`file://` is definitely seven characters...

Reply:
It's zero indexed

Reply:
so... it was just wrong when set to 7?     sheesh.

Reply:
This fixes support for relative and absolute paths

Reply:
Where it didn't before.   Okay.  Thanks.